app/styles/side-menu/side-menu.scss

```diff
@@ -130,10 +139,63 @@
             }
         }
     }
+}
 
-    ul, li{
-        list-style: none;
-        margin: 0;
-        padding: 0;
+.side-menu-footer{
+    margin-bottom: 16px;
+    
+    ul{
+        display: flex;
+        justify-content: center;
+
+        li{
+            margin-right: 8px;
+
+            &:last-child{
+                margin-right: 0;
+            }
+        }
+
+        a{
+            display: inline-block;
+            color: white;
+            padding: 8px 16px;
+            text-decoration: none;
+            font-size: 10pt;
+            
+            i{
+                color: $vert-zera;
+                margin-right: 8px;
+            }
+            
+            &:hover{
+                background: rgba(white, 0.12);
+                i{
+                    color: white;
+                }
+            }
+        }
+    }
+}
+
+@media only screen and (max-width: 720px){
+    .app-side-menu{
+        & > .side-menu-footer{
+            display: none;
+        }
+    }
+
+    .app-layout{
+        & > .side-menu-footer{
+            display: block;
```

Comment:
Did you check this ?

When I tested it on computer (faking size, etc), the links were on the side, and not on the bottom.
Could you confirm it ?

Reply:
@Hartorn The expected behaviour is: 
![ffs2](https://user-images.githubusercontent.com/2766008/33540686-80ea397c-d8cc-11e7-9d54-4afa51503e9f.gif)

Do you have something different than this?